clients/src/main/java/org/apache/kafka/common/metrics/Sensor.java

```diff
@@ -123,8 +123,8 @@ private void checkQuotas(long timeMs) {
                         throw new QuotaViolationException(String.format(
                             "(%s) violated quota. Actual: (%f), Threshold: (%f)",
```

Comment:
@ijuma Thank you for the review. Replaced with single-quote for just the metric name.